mobile/src/reducers/calendar/calendar-events.model.ts

```diff
@@ -5,7 +5,7 @@ import { IntervalTypeConverter } from './interval-type-converter';
 
 export class CalendarEvents {
 
-    constructor(private readonly calendarEvents: CalendarEvent[]) {}
+    constructor(public readonly calendarEvents: CalendarEvent[], public readonly employeeId?: string) {}
```

Comment:
Do we really need ```employeeId``` in CalendarEvents? is it used somewhere else, expect loadCalendarEventsEpic$?

Reply:
It's used in reducer to populate a Map of Employee->Events[] - from here we could get them to present for an employee details screen

Reply:
But you take it from the action. Also, map key is already employeeId (a pair  <employeeId, calendarEvents>)

Reply:
But you take it from the action. Also, map key is already employeeId (pair <employeeId, calendarEvents>)